### PR TITLE
PLU-261: [onboarding-demo-experiment-2]: add backend change to update flow config

### PR DESCRIPTION
**Pull request by m0nggh**
## Problem
New users are confused with how to get started on Plumber, so GGWP v1 is an onboarding demo flow we are experimenting with to provide more help for users to onboard with Plumber.

## Details
Backend
- Modify `updateFlowConfig` graphql query to allow `onFirstLoad` to be modified too

## Tests
- [x] Notification freq still can be updated properly
- [x] `onFirstLoad` can be updated using graphql

### CODE REVIEW

**Comment by m0nggh** [comment]
* **#622** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/622?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#621** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/621?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#620** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/620?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#619** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/619?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#618** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/618?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#617** <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/617?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `develop-v2`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @m0nggh and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/backend/src/graphql/schema.graphql [hunk 1]
```diff
@@ -411,7 +411,8 @@ enum NotificationFrequency {
 
 input UpdateFlowConfigInput {
   id: String!
-  notificationFrequency: NotificationFrequency!
+  notificationFrequency: NotificationFrequency
+  onFirstLoad: Boolean
```
Comment (pregnantboy) [inline packages/backend/src/graphql/schema.graphql]: nit: hasLoadedOnce

File: packages/types/index.d.ts [hunk 2]
```diff
@@ -162,12 +162,19 @@ export interface IFlowConfig {
...
+  demoConfig?: IFlowDemoConfig
 }
 
 export interface IFlowErrorConfig {
   notificationFrequency: 'once_per_day' | 'always'
 }
 
+export interface IFlowDemoConfig {
+  onFirstLoad: boolean
+  isPreCreated: boolean
```
Comment (pregnantboy) [inline packages/types/index.d.ts]: isAutoCreated

**Review by pregnantboy — APPROVED** [review]
some nits

File: packages/backend/src/graphql/schema.graphql (hunk 1 above)
Comment (m0nggh) [reply packages/backend/src/graphql/schema.graphql]: roger boss

Comment (m0nggh) [reply packages/backend/src/graphql/schema.graphql]: ok means im gonna have to negate everything to the opposite

File: packages/types/index.d.ts (hunk 2 above)
Comment (m0nggh) [reply packages/types/index.d.ts]: roger boss

**Pull request merged**